install/Knative-with-GKE.md

```diff
@@ -135,32 +135,56 @@ rerun the command to see the current status.
 > Note: Instead of rerunning the command, you can add `--watch` to the above
   command to view the component's status updates in real time. Use CTRL + C to exit watch mode.
 
-## Installing Knative Serving
+## Installing Knative components
 
-1. Next, we will install [Knative Serving](https://github.com/knative/serving)
-and its dependencies:
+You have the option to install and use only the Knative components that you
+want. You can install only the component of Knative if you need that
+functionality, for example Knative serving is not required to create and run
+builds.
+
+### Installing Knative Serving
+
+1. Run the `kubectl apply` command to install
+   [Knative Serving](https://github.com/knative/serving) and its dependencies:
     ```bash
     kubectl apply -f https://storage.googleapis.com/knative-releases/serving/latest/release.yaml
     ```
-1. Monitor the Knative components, until all of the components show a `STATUS` of
-`Running`:
+1. Monitor the Knative serving components until all of the components show a
+   `STATUS` of `Running`:
     ```bash
     kubectl get pods -n knative-serving
     ```
 
+### Installing Knative build
```

Comment:
I'm not sure we need to update this readme just to mention that you can install build by itself. I think just having `installing-build-component.md` is sufficient for users that only want build; most users coming to Knative at this point probably want all of serving.

Reply:
Information architecture wise, the Knative/install repo stands alone and therefore, users are going to expect to find install info about build here. I also want to avoid copying all of this file (all the prereq's like k8s, istio, and roles). Otherwise `installing-build-component.md` lacks that info (we dont have an end-to-end set of instructions for installing build by itself).

Ive not ramped up on eventing yet but I imagine we might need another section here too, for installing that component?

Reply:
That makes sense. In that case, can we start standardizing on documenting how to set up Knative components against _any_ k8s cluster, e.g., https://github.com/knative/docs/blob/master/install/Knative-with-any-k8s.md

Then at the top of that page we can have links to setup for GKE, AKS, PKS, etc., which can just be links to each provider's own evergreen documentation?

I don't want to have to copy "installing build" into each provider-specific tutorial, and I don't want to have it only documented in the GKE setup doc.

Reply:
Ive created https://github.com/knative/docs/issues/270 to circle back around to creating shareable Knative specific install steps.